src/wymeditor/editor/external-modules-browserify.js

```diff
@@ -5,5 +5,6 @@
 // task.
 WYMeditor.EXTERNAL_MODULES = {
     ObjectHistory: require("object-history"),
-    Combokeys: require("combokeys")
+    Combokeys: require("combokeys"),
+    Edited: require("edited")
 };
```

Comment:
More accurately, this is for the sake of debugging in IE < 11, where there is no source maps support.